block-storage-attacher/tests/e2e/e2e-tests/portworx_secret.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: px-ibm
-  namespace: portworx
+  name: px-etcd-certs
```

Comment:
should this be `px-ibm` and namespace be `portworx`?

Reply:
No those values are right